packages/format-library/src/annotation/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const name = 'core/annotation';
+
+export const annotation = {
```

Comment:
Should we define this in the annotations package instead? I think it makes sense.